### DISCUSSION TITLE
stupid question - sorry in advance. how do i install on desktop pc (not one of with the hardware shown in one click methods)?

### DISCUSSION THREAD

**Discussion by flurb789** [mempool/mempool#3617] [Q&A]
I am running a bitcoin core node just on my normal windows pc.
how do i install a local version of mempool.space? all of the one click options seem to be tailored to umbrel, mynode, etc.
thanks and sorry i'm a moron.

**Comment by softsimon** [comment]
Here is a video guide on how to install it on a Linux desktop

https://www.youtube.com/watch?v=0ctD99DvWa8

**Comment by wiz** [comment]
I’m sure it’s possible to run mempool on windows but we don’t support that. You could probably run it in a VM

**Comment by softsimon** [comment]
**Easy mode:**

Setup `mempool-config.json` in your /backend folder with your Bitcoin Core credentials and BACKEND to `"none"`. Database and Electrum server is optional so you can try without it. But MySQL/MariaDB is recommended.

From the Backend folder:
`npm install`
`npm run build && npm run start `

From the Frontend folder:
`npm install`
`npm run start`

This launches the site running a local dev webserver.
